scripts/SD_converter.py

```diff
@@ -0,0 +1,127 @@
+import json
```

Comment:
Please, move this file to the main folder, so I do not have to always specify file paths with ../